lib/send-mail.ts

```diff
@@ -60,13 +60,14 @@ export function parseMBox(mbox: string, gentle?: boolean):
     let subject: string | undefined;
     let to: string | undefined;
 
-    for (const line of header.split(/\n(?![ \t])/)) {
+    for (let line of header.split(/\n(?![ \t])/)) {
+        line = lines[i].replace("\r\n", "\n").replace("\n", " ");
```

Comment:
This causes plenty of errors in the PR's Checks.

How about doing this instead?

```typescript
    for (const line of header.split(/\n(?![ \t])/)
        .map(e => e.replace("\r\n", " ").replace("\n", " "))) {
```

Reply:
Your project - your style 😄 
Done.

Reply:
..even though the column wrap limit is weird. Are you working with a Unix terminal from the 80ies..?

Reply:
> Are you working with a Unix terminal from the 80ies..?

It is the Git project's requirement, not any hardware's. And personally, I don't believe that I am in the business of questioning successful projects' idiosyncrasies too much, myself 😉 I mean, if I started, the FFMPEG project would lend itself as a splendid target, wouldn't it? 😁

Reply:
Absolutely! I'm in no way standing behind what the ffmpeg project does. I'd have a long list to tell ;-)